appendix-basic65.tex

```diff
@@ -7358,18 +7532,19 @@ \subsection{POINTER}
 \begin{description}[leftmargin=2cm,style=nextline]
 \item [Token:] \$CE \$0A
 \item [Format:] {\bf POINTER(}variable{\bf)}
-\item [Usage:]  Returns the current address of a variable
+\item [Usage:]  The current address of a variable
                 or an array element as a 32-bit pointer.
+
                 For string variables, it is the address of
                 the string descriptor, not the string itself.
-                The string descriptor consists of three bytes
-                (length, string address low, string address high).
+                The string descriptor consists of three bytes:
+                length, string address low, string address high.
```

Comment:
For the info on the string descriptor, should we give an extra hint that string address low/high refer to bank1 (i think?)

Reply:
Excellent catch, added a note.